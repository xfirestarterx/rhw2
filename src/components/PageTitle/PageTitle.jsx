import React from 'react';
import PropTypes from 'prop-types';
import styles from './PageTitle.styl';

const PageTitle = ({ text = 'Default text' }) => <h1 className={styles.PageTitle}>{text}</h1>;

PageTitle.propTypes = {
  text: PropTypes.string.isRequired,
};

export default PageTitle;
