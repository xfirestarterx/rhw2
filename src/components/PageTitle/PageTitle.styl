.PageTitle
    font-size 24px
    text-transform uppercase
    margin-bottom 40px
