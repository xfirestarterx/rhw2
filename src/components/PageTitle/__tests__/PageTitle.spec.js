import React from 'react';
import renderer from 'react-test-renderer';
import PageTitle from '../PageTitle';

describe('<PageTitle> spec', () => {
  it('renders', () => {
    const tree = renderer
    .create(<PageTitle text="title" />)
    .toJSON();

  expect(tree).toMatchSnapshot();
  });
});