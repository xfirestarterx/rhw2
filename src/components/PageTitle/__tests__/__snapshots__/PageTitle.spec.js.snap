// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageTitle> spec renders 1`] = `
<h1
  className="PageTitle"
>
  title
</h1>
`;
